test/mocks/api/v1/mock_resources.proto

```diff
@@ -49,6 +49,7 @@ message MockResource {
     string data = 1 [json_name="data.json", (gogoproto.jsontag) = "data.json"];
     string some_dumb_field = 100 [(core.solo.io.skip_hashing) = true];
 
+    // comments that go above the basic field in our docs=
```

Comment:
might be a stray comment

Reply:
No I believe that comment is there to show how comments in that location wind up in our docs